internal/services/redis/redis_cache_data_source.go

```diff
@@ -235,6 +235,11 @@ func dataSourceRedisCache() *pluginsdk.Resource {
 				Sensitive: true,
 			},
 
+			"access_keys_authentication_disabled": {
```

Comment:
This does not follow our guidelines on property names, specifically for bool. We have a [section in our contributor docs on property names](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-naming.md#property-naming) with the standards that we try to enforce in the provider, so this should be renamed to 
```suggestion
			"access_keys_authentication_enabled": {
```

Reply:
@stephybun the problem with `_enabled` suffix is it does not align with the terminology we use in Azure documentation and REST API:

- https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-azure-active-directory-for-authentication#disable-access-key-authentication-on-your-cache (in portal the checkbox is called `Disable Access Keys Authentication`)
- https://learn.microsoft.com/en-us/rest/api/redis/redis/create?view=rest-redis-2024-03-01&tabs=HTTP (the request body is `properties.disableAccessKeyAuthentication`)

cc: @magodo

Reply:
As discussed internally, we're happy to flip the property to conform to the guideline. Refer to commit 01bff84598.